packages/components/filters/filter-menu/src/filter-menu.tsx

```diff
@@ -34,7 +34,7 @@ export type TFilterMenuProps = {
    * the alternative to a rect element is passing in an array of operators and a handler function,
    * which is more achievable here than for the input in the body, but could still be restrictive
    */
-  operatorsInput?: ReactNode;
+  renderOperatorsInput?: () => ReactNode;
```

Comment:
I think that passing the operators input as a `ReactNode` makes sense here, in large part to minimize the amount of props necessary to achieve the necessary UI.  It forces the consuming application to handle state for the operators completely outside the `Filters` component, which I think is a good thing.

Reply:
Also made it a function that returns a `ReactNode`, which I think we should consider making standard practice when passing props that render child components.